source/icu.net/NativeMethods/NativeMethods.cs

```diff
@@ -563,7 +563,6 @@ private static string GetString(Func<IntPtr, int, Tuple<ErrorCode, int>> lambda,
 					ExceptionFromErrorCode.ThrowIfError(err);
 				if (outLength >= length)
 				{
-					err = ErrorCode.NoErrors; // ignore possible U_BUFFER_OVERFLOW_ERROR or STRING_NOT_TERMINATED_WARNING
```

Comment:
I wonder if it's maybe reasonable/useful to retain the comment.

Reply:
It seems implicit in the preceding `if (err != ErrorCode.BUFFER_OVERFLOW_ERROR && err != ErrorCode.STRING_NOT_TERMINATED_WARNING` and not true in the following `(err, outLength) = lambda(resPtr, length);`.

Reply:
Sorry. I didn't expand the file to be able to see those lines. True.